src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery.Test/ScenarioTests/A2A/A2ATestsHelper.ps1

```diff
@@ -0,0 +1,214 @@
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+########################## Site Recovery Tests #############################
+
+##Default Value ##
+$seed = "98"
+function getVaultName{
+    return "A2APowershellTest" + $seed;
+}
+
+function getVaultRg{
+    return "A2APowershellTestRg" + $seed;
+}
+
+
+function getVaultRgLocation{
+    return "eastus"
+}
+
+function getVaultLocation{
+     return "eastus"
+}
+
+function getPrimaryLocation
+{
+    return "westus"
+}
+
+function getRecoveryLocation{
+  return getVaultLocation
+}
+
+function getPrimaryFabric{
+    return  "a2aPrimaryFabric"+$seed
+
+}
+
+function getRecoveryFabric{
+    return  "a2aRecoveryFabric"+$seed
+
+}
+
+function getAzureVm{
+    param([string]$primaryLocation)
+    
+        $VMLocalAdminUser = "adminUser"
+        $VMLocalAdminSecurePassword = "password"
+        $password=$VMLocalAdminSecurePassword|ConvertTo-SecureString -AsPlainText -Force
+        $Credential = New-Object System.Management.Automation.PSCredential ($VMLocalAdminUser, $password);
+        New-AzureRmVM -Name MyVm -Credential $Credential -location getPrimaryLocation
+}
+
+function getPrimaryPolicy{
+    return "TestA2APolicy1" + $seed;
+}
+
+function getRecoveryPolicy{
+    return "TestA2APolicy1" + $seed;
+}
+
+function getPrimaryContainer{
+    return "A2APrimaryContainer"+ $seed;
+}
+
+
+function getRecoveryContainer{
+    return "A2ARecoveryContainer"+ $seed;
+}
+
+
+function getPrimaryContainerMapping{
+    return "A2APCM"+ $seed;
+}
+
+
+function getRecoveryContainerMapping{
+    return "A2ARCM"+ $seed;
+}
+
+function getPrimaryNetworkMapping{
+    return "A2ANetworkMapping"+ $seed;
+}
+
+function getRecoveryNetworkMapping{
+    return "A2ARecoveryNetworkMapping"+ $seed;
+}
+
+function getPrimaryNetworkId{
+    param([string] $location , [string] $resourceGroup)
+
+    
+}
+
+function getRecoveryNetworkId{
+    param([string] $location , [string] $resourceGroup)
+
+    $primaryNetworkName = "recoveryNetwork"+ $location + $seed;
+    $virtualNetwork = New-AzureRmVirtualNetwork `
+          -ResourceGroupName $resourceGroup `
+          -Location $location `
+          -Name $primaryNetworkName `
+          -AddressPrefix 10.0.0.0/16
+    $virtualNetwork.id
+}
+
+
+##
+
+<#
+.SYNOPSIS
+Wait for job completion
+Usage:
+    WaitForJobCompletion -JobId $Job.ID
+    WaitForJobCompletion -JobId $Job.ID -NumOfSecondsToWait 10
+#>
+function WaitForJobCompletion
+{ 
+    param(
+        [string] $JobId,
+        [int] $JobQueryWaitTimeInSeconds = 20,
+        [string] $Message = "NA"
+        )
+        $isJobLeftForProcessing = $true;
+        do
+        {
+            $Job = Get-AzureRmRecoveryServicesAsrJob -Name $JobId
+            Write-Host $("Job Status:") -ForegroundColor Green
+            $Job
+
+            if($Job.State -eq "InProgress" -or $Job.State -eq "NotStarted")
```

Comment:
If statements require boolean conditions. So, assigning a boolean based on a boolean is redundant. It can simply be:
```powershell
$isJobLeftForProcessing = ($Job.State -eq 'InProgress' -or $Job.State -eq 'NotStarted')
```
Also, double quotes are used for interpolated strings (strings that can substitute variables in-place) where single quotes are used for string literals.